presto-hive/src/main/java/com/facebook/presto/hive/HiveUtil.java

```diff
@@ -1022,7 +1025,10 @@ public static Optional<String> getPrefilledColumnValue(HiveColumnHandle columnHa
         if (isFileModifiedTimeColumnHandle(columnHandle)) {
             return Optional.of(String.valueOf(fileSplit.getFileModifiedTime()));
         }
-
+        if (isRowIdColumnHandle(columnHandle)) {
+            // TODO handle ROW_ID here; maybe just the roiwIdPartitionComponent
```

Comment:
better would be not to remove this code and not use the prefilled column type as it's not relevant here.

Reply:
I'll see if I can handle more in this PR